src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,43 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static void getStatistic(String fromFileName, String toFileName) {
+        File fileFrom = new File(fromFileName);
+        File fileTo = new File(toFileName);
+        StringBuilder returnedStringToFile = new StringBuilder();
+        int supply = 0;
+        int buy = 0;
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileFrom))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                String[] stringsFromLine = value.split(",");
```

Comment:
0, 1, and "," + "supply" are magic numbers / string, let's declare them as constants